code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -124,6 +124,17 @@
 	bitesize = 4
 	list_reagents = list("sugar" = 15)
 
+/obj/item/reagent_containers/food/snacks/candy/plasmapuffs
+	name = "plasma puffs"
+	desc = "A savory snack made from a mixture of dehydrated plasma and various spices!"
+	icon_state = "plasmapuffs"
+	trash = /obj/item/trash/plasmapuffs
+	filling_color = "#FFFFFF"
+	bitesize = 4
+	list_reagents = list("plasma" = 0.1, "sugar" = 20)
+	tastes = list("plasma" = 1)
+
```

Comment:
Sorry for these pester reviews, just for consistency with the rest of the file. If you wouldn't mind removing the extra spaces around your edits in the rest of the file it'd be nice
```suggestion
```